docs/rfc/block-registration.md

```diff
@@ -69,19 +69,32 @@ To register a new block type, start by creating a `block.json` file. This file:
 	"icon": "star",
 	"description": "Shows warning, error or success notices  ...",
 	"keywords": [ "alert", "message" ],
+	"textDomain": "my-plugin",
 	"attributes": {
 		"message": {
 			"type": "string",
 			"source": "html",
 			"selector": ".meessage"
 		}
 	},
-	"edit": "blocks/notice-edit.js",
-	"save": "blocks/notice-save.js",
 	"styles": [ 
 		{ "name": "default", "label": "Default", "isDefault": true }, 
 		{ "name": "other", "label": "Other" }
-	]
+	],
+	"editorScript": "file:build/editor.assets.json",
+	"script": {
```

Comment:
The difference between script and editorScript is not obvious, is this more "frontendScript" or a script that is loaded everywhere?

Should we support "file:build/editor.assets.json" from the start? Can this be left out and considered later as needed? It does seem unnecessary at the moment.

Reply:
Well, it's not clear for me as well. According to PHP code comments, it loads on both frontend and in the editor. I'm sure it's true for `style`, it has to be confirmed for `script` 🤷‍♂

Reply:
https://github.com/WordPress/wordpress-develop/blob/14066540c00dc82a7682e016f35861a356a555ab/src/wp-includes/script-loader.php#L2735-L2752

It's both the editor and frontend.